src/client/src/components/TopNavBarLink/index.tsx

```diff
@@ -39,20 +39,34 @@ const TopNavBarLink = ({
       onClick={handleClick}
       className={styles.container}
       onKeyUp={keyUpHandler}
+      aria-label={
+        visitedCheck || isSelected
```

Comment:
should we have `(..)`  parenthesis here?

Reply:
Honestly I would extract some of this out into a const function. It is kinda hard to read/follow.

Reply:
@annielovesjs

Reply:
@jcoc611-microsoft the parenthesis gets automatically removed

Reply:
@SahilTara good point! I just abstracted it to a function call now so it should be more readable